style/roads.mss

```diff
@@ -2658,6 +2691,16 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       [zoom >= 12] {
         line-width: 2;
         line-color: #999;
+        [preserved = 'yes'][zoom >= 15] {
+          #roads-fill, #bridges {
+            dark/line-width: 3;
+            dark/line-color: #999999;
```

Comment:
You should specify the same color in the same way, not sometimes with `#999`, sometimes `#999999`.